raiden/tests/integration/api/test_pythonapi.py

```diff
@@ -844,3 +845,22 @@ def test_raidenapi_channel_lifecycle(
         },
         retry_timeout,
     )
+
+
+@raise_on_failure
+@pytest.mark.parametrize("number_of_nodes", [2])
+@pytest.mark.parametrize("channels_per_node", [1])
+def test_same_addresses_for_payment(raiden_network, token_addresses):
+    app0, _ = raiden_network
+    api0 = RaidenAPI(app0.raiden)
+    registry_address = app0.raiden.default_registry.address
+    token_address = token_addresses[0]
+    address_of_app0 = app0.raiden.address
```

Comment:
I don't think `address_of_app0` provides any benefit over `app0.raiden.address` and would just use the latter directly.

Reply:
good point

Reply:
good point